src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -99,15 +104,13 @@ public Response getUsers(@Auth AuthUser authUser, @PathParam("roleName") String
   @Produces("application/json")
   @PermitAll
   @Timed
-  public Response getUser(@Auth AuthUser authUser) {
+  public Response getUser(@Auth DuosUser duosUser) {
     try {
-      User user = userService.findUserByEmail(authUser.getEmail());
-      if (Objects.isNull(authUser.getUserStatusInfo())) {
-        samService.asyncPostRegistrationInfo(authUser);
+      if (Objects.isNull(duosUser.getUserStatusInfo())) {
+        samService.asyncPostRegistrationInfo(duosUser);
       }
-      JsonObject userJson = userService.findUserWithPropertiesByIdAsJsonObject(authUser,
-          user.getUserId());
-      return Response.ok(gson.toJson(userJson)).build();
+      User syncedUser = nihService.syncAccount(duosUser);
```

Comment:
This is the meat of this PR. Previously, we didn't have a method to get a user with all properties and library cards which is why we have the `userService.findUserWithPropertiesByIdAsJsonObject` method. When we added `nihService.syncAccount`, we added a new DAO call to do that population at the JDBI level instead of in service layer code.